### DISCUSSION TITLE
View not accessible from api docs

### DISCUSSION THREAD

**Discussion by tbcomputerguy** [supabase/supabase#18902] [Questions] [answered]
Hello, first time using Supabase.  Setup a view from my table and all works except no access from the api...the page crashes giving me this error!  **Application error: a client-side exception has occurred (see the browser console for more information).**

Any help would be greatly appreciated.

**Comment by GaryAustin1** [comment, marked as answer]
A view should look like a table to the API.  Are you getting no error info?  Any errors in the dashboard Postgres logs?  Does the request show up in the dashboard API Edge logs?
